app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
```

Comment:
The `wife` and `husband` attributes should be initialized in the `__init__` method to avoid `AttributeError` when they are accessed before being set. Consider initializing them as `None`.